pkgs/development/python-modules/objax/default.nix

```diff
@@ -40,6 +51,23 @@ buildPythonPackage rec {
     "objax"
   ];
 
+  catchConflicts = false;
+
+  nativeCheckInputs = [
+    keras
+    pytestCheckHook
+    tensorflow
+  ];
+
+  pytestFlagsArray = [
+    "tests/*.py"
+  ];
```

Comment:
why is this necessary? i would imagine it ought to default to something like this?

Reply:
I thought too, but it is not the case (it fails to collect any test otherwise).
The script that they use to run the tests does the same thing: https://github.com/google/objax/blob/master/tests/run_tests.sh#L23

Reply:
Huh, weird

Reply:
The file names are so bad. This is the only way to do it.
https://docs.pytest.org/en/7.2.x/explanation/goodpractices.html#conventions-for-python-test-discovery